<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simpleEditDialog</class>
 <widget class="QDialog" name="simpleEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SwarmTv Simple Filter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="swarmtvqtresources.qrc">
    <normaloff>:/swarmtv/mainlogo/resources/swarmtv_logo.png</normaloff>:/swarmtv/mainlogo/resources/swarmtv_logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Simple filter Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Title regexp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Exclude regexp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Category</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>source regexp</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="nameSimpleLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="titleSimpleLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="excludeSimpleLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="categorySimpleLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="sourceSimpleLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Min size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="minSizeSimpleLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Max size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="maxSizeSimpleLineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="guessSEPushButton">
           <property name="text">
            <string>Guess season and episode</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>From episode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="episodeSpinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>From season</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="seasonSpinBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>No duplicate Filter type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="nodupSimpleComboBox">
         <item>
          <property name="text">
           <string>newer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>newer-title</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>link</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>unique</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>unique-title</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>none</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="testPushButton">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="simpleButtonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>label_8</zorder>
 </widget>
 <tabstops>
  <tabstop>nameSimpleLineEdit</tabstop>
  <tabstop>titleSimpleLineEdit</tabstop>
  <tabstop>excludeSimpleLineEdit</tabstop>
  <tabstop>categorySimpleLineEdit</tabstop>
  <tabstop>sourceSimpleLineEdit</tabstop>
  <tabstop>minSizeSimpleLineEdit</tabstop>
  <tabstop>maxSizeSimpleLineEdit</tabstop>
  <tabstop>nodupSimpleComboBox</tabstop>
  <tabstop>simpleButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="swarmtvqtresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>simpleButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>simpleEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
